dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestDataInterface.cpp

```diff
@@ -1374,20 +1374,13 @@ TEST(DataInterfaceIIDMTest, testStaticVarCompensatorIIDM) {
   svc->setValue(StaticVarCompensatorInterfaceIIDM::VAR_P, 4.);
   svc->setValue(StaticVarCompensatorInterfaceIIDM::VAR_Q, 1);
   svc->setValue(StaticVarCompensatorInterfaceIIDM::VAR_STATE, OPEN);
-  svc->setValue(StaticVarCompensatorInterfaceIIDM::VAR_REGULATINGMODE, StaticVarCompensatorInterface::RUNNING_V);
```

Comment:
WOould it be possible to add a test with a standByAutomaton? Maybe in private repo?

Reply:
yes, I think it is a good idea, but it must be on private repo as private extension